bubblesub/ui/util.py

```diff
@@ -28,15 +28,20 @@
 
 SUBS_FILE_FILTER = "Advanced Substation Alpha (*.ass)"
 VIDEO_FILE_FILTER = "Video filters (*.avi *.mkv *.webm *.mp4);;All files (*.*)"
-AUDIO_FILE_FILTER = "Audio filters (*.wav *.mp3 *.flac *.avi *.mkv *.webm *.mp4);;All files (*.*)"
+AUDIO_FILE_FILTER = (
+    "Audio filters"
+    + "("
+    + "*.wav *.mp3 *.flac *.avi *.mkv *.webm *.mp4"
+    + ");;All files (*.*)"
```

Comment:
You don't need to use `+` for concatenation of string literals in Python, and putting opening brace in a separate line looks kinda dumb

Reply:
Just following some indentation specs, need your help, how would you change that?

Reply:
What do you think about this?

```python3
AUDIO_FILE_FILTER = (
    "Audio filters (*.wav *.mp3 *.flac *.avi *.mkv *.webm *.mp4);;"
    "All files (*.*)"
)
```

Reply:
I agree :)